tests/src/test/scala/common/Wsk.scala

```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```

Comment:
That should be an `Option[File]` with a default value of `None`.

Then to use it against the Java API you can use `stdinFile.getOrElse(null)`. At least keeps the `null` out of the Scala API.